packages/dso-toolkit/src/components/plekinfo-card/plekinfo-card.variables.scss

```diff
@@ -0,0 +1,12 @@
+@use "../../variables/colors";
+
```

Comment:
@hansgrimm Wat vindt je van de kleuren?

Reply:
Zijn aangepast cf. commentaar van Django in het issue https://github.com/dso-toolkit/dso-toolkit/issues/2797#issuecomment-2396409855